sentry-ruby/lib/sentry/cron/monitor_check_ins.rb

```diff
@@ -57,8 +67,6 @@ def sentry_monitor_config
         end
       end
 
-      extend ClassMethods
```

Comment:
this was because I was earlier trying to make something that could be injected to both classes and modules, but now it's class only so it's not longer needed.